src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    }
+
+    if (action.type === 'removeProperties') {
+      for (const i of action.keysToRemove) {
```

Comment:
Let's create more meaningful variable name
```suggestion
      for (const key of action.keysToRemove) {
```